tests/containers/multi_runtime.pm

```diff
@@ -0,0 +1,179 @@
+# SUSE's openQA tests
+#
+# Copyright 2024-2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate docker and podman network interoperability
+# - validate if podman and docker networking works together
+# - check if firewalld doesn't break either
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base qw(containers::basetest);
+use testapi;
+use serial_terminal qw(select_serial_terminal select_user_serial_terminal);
+use utils;
+use containers::common qw(install_packages);
+use Utils::Logging 'save_and_upload_log';
+
+my $port = 8000;
+
+sub run_tests {
+    my $ip_version = shift;
+
+    my $test_image = "registry.opensuse.org/opensuse/nginx";
+    my $test_url = "https://registry.opensuse.org/v2/";
```

Comment:
Why not simply conncheck.opensuse.org? That means we're not hammering on the actual container registry.